client/app/(authenticated)/dashboard/profile/page.tsx

```diff
@@ -8,8 +8,7 @@ import { authOptions } from "@/app/api/auth/[...nextauth]/route";
 export default async function Page() {
   const session = await getServerSession(authOptions);
   const appRole = session?.user?.app_role;
-  const isCasInternal =
-    appRole?.includes("cas") && !appRole?.includes("pending");
+  const isCasInternal = appRole?.includes("cas");
```

Comment:
Suggest using `const isCasInternal = session?.identity_provider?.includes("idir");` because ` const isCasInternal = appRole?.includes("cas");` will fail if IDIR user is new to the system as the new user has no role.

Reply:
Thanks!!

Reply:
Seems there is an enum for identity provider; so,probably better to use:
`IDP.IDIR`